styles.css

```diff
@@ -11,8 +11,8 @@ a {
 }
 
 body {
-  font-family: 'Poppins', sans-serif;
-  height: 5410px;
+  font-family: 'Inter', sans-serif;
+  height: 6351px;
   align-items: center;
   width: 375px;
```

Comment:
- [OPTIONAL] As our design should be responsive from 375px to 768px as per requriements, giving exact width of 375px to body tag goes against responsive design. kindly consider removing this line and also consider removing all the lines where you are giving your containers exact widths and rely on paddings, margins and flex,grid for layout.